lib/porky_lib/file_service.rb

```diff
@@ -200,7 +160,7 @@ def s3_client
     @s3_client ||= Aws::S3::Client.new
   end
 
-  def s3
-    @s3 ||= Aws::S3::Resource.new
+  def input_invalid?(file, bucket_name, key_id)
```

Comment:
shouldn't this go in the `file_service_helper.rb` too? I see it's being used on the `unencrypted/file_service.rb `

Reply:
It could be, its just that they have a different number of params (I tried overloading a method, but that's not supported in rails)